tests/snmp/test_snmp_memory.py

```diff
@@ -142,6 +144,31 @@ def test_snmp_memory_load(duthosts, enum_rand_one_per_hwsku_hostname, localhost,
                   "sysTotalFreeMemory differs by more than {}".format(percentage))
 
 
+def test_snmp_avail_memory_load(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_all_duts, load_memory):
```

Comment:
Sorry, one basic question, just want to know why is the "load_memory" function necessary here? thanks

Reply:
load_memory is starting a python script which prints the empty character string set to a specific length.
The memory usage increases when this script is running